packages/optimizer/lib/transformers/ServerSideRendering.js

```diff
@@ -85,7 +85,7 @@ class ServerSideRendering {
       // amp-experiment is a render delaying extension iff the tag is used in
       // the doc. We check for that here rather than checking for the existence
       // of the amp-experiment script in IsRenderDelayingExtension below.
-      if (node.tagName === 'amp-experiment') {
+      if (node.tagName === 'amp-experiment' && !this.isAmpExperimentUsed(node)) {
```

Comment:
The logic is the wrong way around:
```suggestion
      if (node.tagName === 'amp-experiment' && this.isAmpExperimentUsed(node)) {
```

Reply:
done